docs/tutorial/step2_parsing.md

```diff
@@ -138,10 +136,13 @@ this.atomicExpression =
 
 #### How can the Parser be debugged? 
 The grammar rules above do not only define the grammar, they are also the code that will be run
-during parsing. This means that debugging the parser **simply means adding a break
-point in the grammar**. There **do not** exist two different representations for for the grammar
-and the runnable implementation (for example grammar file vs generated code in the case of parser generators).
+during parsing. This means that you can debug the parser **simply by adding a break
+point in the grammar**.
 
+There **do not** exist two different representations for the grammar
+and the runnable implementation (for example, grammar file vs generated code in the case of parser generators). Again, please note that Chevrotain is **NOT** a parser generator.
+
+The reasoning behind this decision is [explained here[(https://www.bountysource.com/issues/38668050-chevrotain-as-the-generated-code-from-peg).
```

Comment:
This link may be too much details and too deep a discussion
for tutorial level. 
It is also just a link to one of the issues in this repository so seems strange to link to bounty Source.

If you want to link to some extra info here use the FAQ instead:

https://github.com/SAP/chevrotain/blob/master/docs/faq.md#-why-should-i-use-a-parsing-dsl-instead-of-a-parser-generator